spec.html

```diff
@@ -936,6 +989,24 @@ <h1>The AsyncContext Object</h1>
 
     <emu-clause id="sec-asynccontext-abstract-operations">
       <h1>AsyncContext Abstract Operations</h1>
+
+      <emu-clause id="sec-gethostdefinedasynccontextmapping" type="abstract operation">
+        <h1>GetHostDefinedAsyncContextMapping ( ): a List of Async Context Mapping Records</h1>
```

Comment:
```suggestion
        <h1>HostGetTopLevelAsyncContextMapping ( ): a List of Async Context Mapping Records</h1>
```

All host operations names start with `Host*`

Reply:
Updated! (there is an exception like [InitializeHostDefinedRealm](https://tc39.es/ecma262/#sec-initializehostdefinedrealm) though).

Reply:
InitializeHostDefinedRealm's algorithm is defined in ECMA-262, not in HTML :)